components/Dashboard/TransactionTable.vue

```diff
@@ -64,19 +273,74 @@ export default {
     await axios
       .get("https://pwcdemo-1c4d3.firebaseio.com/users.json")
       .then(response => {
-        (this.loading = false), (this.facts = Object.values(response.data));
+        const postArray = [];
+        for (const key in response.data) {
+          postArray.push({ ...response.data[key], id: key });
+        }
+        (this.loading = false), (this.facts = postArray);
+        // console.log(this.facts);
+
+        // console.log(this.facts[0]);
       });
   },
   methods: {
+    fetchDataFromTable(index, factID) {
+      this.sizeFormT = this.facts[index];
+      //    this.sizeFormT = this.sizeForm;
+      this.jsonID = this.facts[index].id;
+    },
+    UpdateForm() {
+      this.$refs.sizeFormT.validate(valid => {
+        if (valid) {
+          axios
+            .put(
+              "https://pwcdemo-1c4d3.firebaseio.com/users/" +
+                this.jsonID +
+                ".json",
+              this.sizeFormT
+            )
+            .then(response => {
+              this.$message({
+                message: "Congrats, This user has been updated.",
+                type: "success"
+              });
+              this.dialogVisible = false;
+            })
+            .catch(error => {
+              this.$message({
+                message: error,
+                type: "error"
+              });
+            });
+        }
+      });
+    },
     handleClose(done) {
       this.$confirm("Are you sure to close this dialog?")
         .then(_ => {
           done();
         })
         .catch(_ => {});
     },
-    deleteRow(index, facts) {
-      facts.splice(index, 1);
+    deleteRow(index, factID) {
+      console.log("index: ", factID);
+      axios
+        .delete(
+          "https://pwcdemo-1c4d3.firebaseio.com/users/" + factID + ".json"
```

Comment:
you may use template literals here.

Reply:
I used es6 template. sorry I missed this issue.